src/controllers/channel.controller.js

```diff
@@ -58,3 +58,14 @@ exports.updateChannel = async (req, res) => {
         return response_500(res, "Error updating channel", err)
     }
 }
+
+exports.deleteChannel = async (req, res) => {
+    try {
+        const channelId = parseInt(req.params.id);
```

Comment:
channel object is available in req.channel

Reply:
req.body.channel if I understand correctly

Reply:
anyway added that, kindly review now